bench/app.py

```diff
@@ -79,39 +93,37 @@ def setup_details(self):
 			self.on_disk = True
 			self._setup_details_from_mounted_disk()
 
-		# fetch meta for repo from remote git server - traditional get-app url
-		elif is_git_url(self.name):
-			if self.name.startswith("git@") or self.name.startswith("ssh://"):
-				self.use_ssh = True
-			self._setup_details_from_git_url()
+		# fetch meta for repo from git url - traditional get-app url
+		# NOTE: Adding a check for owner here for making `organisation/repoository` format work
+		elif parsed_git_obj and parsed_git_obj.owner:
+			self.tag = self.branch
+			self.org = parsed_git_obj.owner
+			self.repo = parsed_git_obj.name
```

Comment:
We could also set `self.use_ssh` if the passed name was a git SSH url?

Reply:
it's not necessary i think - it's there to go with the partial url format (when user provides `--use-ssh` - maybe we can change the naming a bit as it definitely is a bit confusing as to with what it might be used [maybe `use_ssh_for_partial_url`?])